### DISCUSSION TITLE
New to the tool.

### DISCUSSION THREAD

**Discussion by ravzkie12** [n00mkrad/flowframes#31] [General]
Hello, First time using the software.

My Footage seems to be shorter than the original video. Is this normal or did I do something wrong?
![image](https://user-images.githubusercontent.com/26573736/110031232-9afd4c80-7d71-11eb-9411-c961fdd3c809.png)

**Comment by n00mkrad** [comment]
Which version are you on?

**Reply by ravzkie12** [reply]
FlowframesApp1.22.3, Non-Patreon.
P.S Don't worry about the quality of the screenshot. the conversion is good but the clip is shorten.

**Reply by n00mkrad** [reply]
Can you reproduce this and then send me logs (located at FlowframesData/logs/)?

**Reply by ravzkie12** [reply]
He's all the logs sir. And still same result.


[rife-ncnn-log.txt](https://github.com/n00mkrad/flowframes/files/6091950/rife-ncnn-log.txt)
[sessionlog.txt](https://github.com/n00mkrad/flowframes/files/6091951/sessionlog.txt)
[ffmpeg.txt](https://github.com/n00mkrad/flowframes/files/6091952/ffmpeg.txt)

**Comment by n00mkrad** [comment]
Why do you use NCNN?

CUDA should be ~2x faster.

**Reply by ravzkie12** [reply]
Ah, Sorry sir.
So if I choose CUDA, my issue will be fixed?

**Reply by ravzkie12** [reply]
It Keeps getting an error. Misaligned address once

**Reply by n00mkrad** [reply]
Disable any kind of overclock if you have one applied

**Reply by ravzkie12** [reply]
Its all good but the clip shorten problem is still there.

**Reply by ravzkie12** [reply]
Its like the video speed up. the video's original fps is 24. so I change the settings to 30 input fps x2. Because I want it to be 60
![image](https://user-images.githubusercontent.com/26573736/110174809-ab79f980-7e3b-11eb-954b-4b78a0e54f5a.png)
